core/Test/Tasty/Providers.hs

```diff
@@ -37,4 +40,13 @@ testFailed desc = Result
   , resultDescription = desc
   , resultShortDescription = "FAIL"
   , resultTime = 0
+  , printResultDetails = Nothing
```

Comment:
Instance of https://github.com/mbj/tasty/pull/1/commits/f4a22fbcbf2861adca73bf9f30488e47d64361c3#r337588780

Reply:
See: https://github.com/mbj/tasty/pull/1#discussion_r337602364